Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM ruby:2.7.2
+
+RUN apt-get update -qq && apt-get upgrade -y
+RUN apt-get install -y build-essential nodejs && apt-get clean
+RUN gem install foreman
+
+ENV GOVUK_APP_NAME authenticating-proxy
+ENV PORT 3107
+ENV RAILS_ENV production
+
+ENV APP_HOME /app
+RUN mkdir $APP_HOME
+
+WORKDIR $APP_HOME
+ADD Gemfile* $APP_HOME/
+ADD .ruby-version $APP_HOME/
```

Comment:
Better to use `COPY` rather than the mostly-legacy `ADD` (https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#add-or-copy) - and similarly below. (I know this is only meant to be a rough thing, but might as well follow best practices to set a good example.)